packages/building-blocks/src/stories/ColorPalette.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import styled from "styled-components";
+import { getMostReadableTextColor } from "@opentripplanner/core-utils/lib/route";
+
+interface Hue {
```

Comment:
This type can probably be exported and then used as the type in the different color files.